{"name": "not authenticated user -- @1.12 cases", "status": "passed", "steps": [{"name": "Given alien makes get request: /reconciliation/services_compensations/", "status": "passed", "start": 1610701820598, "stop": 1610701820624}, {"name": "Then check response of request: Authentication credentials were not provided. is True", "status": "passed", "start": 1610701820625, "stop": 1610701820625}], "parameters": [{"name": "url", "value": "/reconciliation/services_compensations/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "True"}], "start": 1610701820597, "stop": 1610701820625, "uuid": "621498d9-37e4-4c3c-9a2b-457c1b261ff9", "historyId": "7d966bca3d43e7928063c8ea050cd1d9", "fullName": "features/III-reconciliation_access.feature:not authenticated user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}